### PR TITLE
basic question, rules, results page done

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Added homepage, rules, results page

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    color: var(--white);
+}
+.navbar{
+    padding: 2rem;
+    color: var(--white);
+    text-shadow: 2px 2px 2px var(--primary-color);
+    text-align: center;
+}
+.heading{
+    font-size: 32px;
```
Comment (shaksham08) [inline css/styles.css]: You can use rem

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+.quiz--poster{
+    width: 100%;
+    height: 13rem;
+    border-radius: 5px 5px 0 0;
+}
+.quiz--content{
+    padding: 0.5rem 1rem;
+}
+.quiz--heading{
+    font-size: 24px;
```
Comment (shaksham08) [inline css/styles.css]: Here also rem

**Review by shaksham08 — APPROVED** [review]
Added one comment , other than all looks good

File: css/styles.css (hunk 2 above)
Comment (bhargaviRdy) [reply css/styles.css]: made changes, thanks for pointing out

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+.quiz--btn{
+    text-decoration: none;
+    display: inline-block;
+    margin: 1rem 0;
+    padding: 0.5rem 1rem;
+    background-color: transparent;
+    border-radius: 0.3rem;
+    border: none;
+    text-align: center;
+    outline: none;
```
Comment (harshitadaswani) [inline css/styles.css]: use outline: 1px solid transparent instead

**Review by harshitadaswani — APPROVED** [review]
The class names are good! Just look at the comment, make changes and merge the pr

File: css/styles.css (hunk 3 above)
Comment (bhargaviRdy) [reply css/styles.css]: sure, thanks for the suggestion

**Pull request merged**